3.5-windowsservercore-1803/sdk/Dockerfile

```diff
@@ -32,17 +32,28 @@ ENV ROSLYN_COMPILER_LOCATION "C:\Program Files (x86)\Microsoft Visual Studio\201
 
 # ngen assemblies queued by VS installers - must be done in cmd shell to avoid access issues
 SHELL ["cmd", "/S", "/C"]
-RUN \Windows\Microsoft.NET\Framework64\v4.0.30319\ngen update `
+
```

Comment:
Maybe it is just my preference, but I don't like this space after the SHELL instruction and the corresponding whitespace before the 'restore' SHELL instruction.